src/components/twilio-voice-basic-call-control/public/twilio-voice-basic-call-control.js

```diff
@@ -189,7 +217,7 @@ class TwilioVoiceBasicCallControl extends HTMLElement {
       : 'none';
   }
 
-  async #updateConferenceCall(callSid, params) {
+  async #updateConference(callSid, params) {
     return await fetch(
```

Comment:
This may be out of scope, but using `return await` inside an `async` function can be unnecessary

Reply:
I believe I used a `return` here because I use the response for error handling here: https://github.com/twilio/twilio-voice-js-reference-components/blob/main/src/components/twilio-voice-basic-call-control/public/twilio-voice-basic-call-control.js#L67

Wdyt?

Reply:
Thanks for considering this small update